internal/service/globalclusterconfig/resource_global_cluster_config.go

```diff
@@ -93,6 +94,11 @@ func Resource() *schema.Resource {
 				},
 			},
 			"custom_zone_mapping": {
+				Deprecated: fmt.Sprintf(constant.DeprecationParamByDateWithReplacement, "1.23.0", "custom_zone_mapping_zone_id"),
```

Comment:
same above https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2637#discussion_r1776788800

Reply:
changed in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2637/commits/a5c40a9437f15380d32824d3d8befe1dd4fbff01